lib/tasks/courses_wikis.rake

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+namespace :courses_wikis do
+  desc 'Creates CoursesWikis association for existing courses through revisions'
+  task migrate: :environment do
+    Rails.logger.debug 'Creating CoursesWikis associations'
+    Course.all.each do |course|
+      course.wikis.push(Wiki.find(course.wiki_ids))
```

Comment:
This needs to be idempotent. Currently, it will fail (because of the uniqueness constraint) if it is run when there are already some CoursesWikis in the DB or if it is run more than once.